### DISCUSSION TITLE
Feature Request - RTL Layout

### DISCUSSION THREAD

**Discussion by stcsdmm7** [primefaces/community#417] [PrimeNG]
Dear team,

I am writing to ask if it is possible to add the feature RTL layout for Verona Layout and Apollo Layout as RTL layout is essential for users who use Arabic language.

Also, we hope you can add the horizontal menu type to Verona Layout and E-commerce into the Poseidon layout.

We appreciate your support and looking forward to hearing soon.

Best regards.

**Comment by stcsdmm7** [comment]
We need to purchase it but it should support RTL, I appreciate your reply if it possible or not

**Comment by cetincakiroglu** [comment]
Hi,

Thanks a lot for sharing feedback and the feature idea. Right now only [Poseidon template](https://primefaces.org/poseidon-ng/#/)  supports RTL but RTL support is in our backlog for PrimeNG Core and we will implement it when the time comes.

**Reply by stcsdmm7** [reply]
Hi, thanks for your reply. Can you add pages E-COMMERCE (product overview, shopping cart, .. ) and APPS (Blog, ..) into Poseidon template (https://primefaces.org/poseidon-ng/#/)?

**Comment by stcsdmm7** [comment]
Hi, thanks for your reply. Can you add pages E-COMMERCE (product overview, shopping cart, .. ) and APPS (Blog, ..) into Poseidon template (https://primefaces.org/poseidon-ng/#/)?

**Comment by stcsdmm7** [comment]
Hi,

Is there any chance to add pages E-COMMERCE (product overview, shopping cart, .. ) and APPS (Blog, ..) into Poseidon template
